presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -50,6 +50,7 @@
 import com.facebook.presto.sql.planner.plan.JoinNode;
 import com.facebook.presto.sql.planner.plan.LateralJoinNode;
 import com.facebook.presto.sql.planner.plan.MergeJoinNode;
+import com.facebook.presto.sql.planner.plan.MetadataDeleteNode;
```

Comment:
May I know the purpose of the changes in this file? Looks like for now the changes are unused. Are you preparing any changes on the UI side?

Reply:
When query state changes, QueryMonitor would invoke method `createGraphvizQueryPlan` to get a 'graphvizPlan' for queryMetadata. If we do not implement the method for MetadataDeleteNode here, it will always print error message like '...Node com.facebook.presto.sql.planner.plan.MetadataDeleteNode does not have a Graphviz visitor...' when we try to execute metadata delete.

Reply:
Thanks for the clarification.